docker-compose.yaml

```diff
@@ -18,7 +18,7 @@ services:
       LIMIT_RESOURCES: 'true'
 
   semantic-search:
-    image: ghcr.io/janeliascicomp/gpt-semantic-search-web:0.1.4
+    image: ghcr.io/janeliascicomp/gpt-semantic-search-web:0.1.5
```

Comment:
Adding caching might be considered a new feature, and then according to [SemVer](https://semver.org/), we should increment the minor version. What do you think?